rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+    def initialize(book,person,date)
+      @book = book
+      book.rentals << self unless  book.rentals.include?(self)
+      
+      @person = person
+      person.rentals << self unless  person.rentals.include?(self)
+      
+      @date = date
+    end
```

Comment:
- [ ] Good job so far 👍 Kindly add the `attr_accessor` for date, person, and book to the `Rental` class as we want to be able to access that data 😄